event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/eventstream/EventStreamJdbcRepository.java

```diff
@@ -128,6 +120,17 @@ public void delete(final UUID streamId) {
         }
     }
 
+    public Stream<EventStreamObject> findAllGreaterThanAndEqualToSequenceNumber(final long sequenceNumber) {
```

Comment:
Should this be findWithSequenceNumberFrom()  ? or something similar

Reply:
changed